src/Core/Session.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace TrendingRepos\Core;
+
+class Session {
+    public function __construct()
+    {
+        if($this->isStarted())
+        { 
+            session_start(); 
+        } 
+    }
+
+    public function isStarted(): bool
+    {
+        return session_status() === PHP_SESSION_ACTIVE;
+    }
+
+    public function set(string $key, string $value)
+    {
+        $_SESSION[$key] = $value;
+    }
+
+    public function get(string $key) 
+    {
+        return $this->has($key) ? $_SESSION[$key] : null;
+    }
+
+    public function all()
+    {
+        return $_SESSION;
+    }
+
+    public function has(string $key): bool
+    {
+        if (empty($_SESSION)) {
+            return false;
+        }
+        return array_key_exists($key, $_SESSION);
+    }
+
+    public function destroy(string $key) 
+    {
+        if($this->has($key)) {
```

Comment:
It would be better if you negate the condition and return if there's no such key. This way the main purpose of the method will be in its body, not inside `if` block.